README.md

```diff
@@ -7,10 +7,11 @@ iocage is a jail/container manager amalgamating some of the best features and
 technologies the FreeBSD operating system has to offer. It is geared for ease
  of use with a simple and easy to understand command syntax.
 
-iocage is in the FreeBSD ports tree as sysutils/iocage.
-To install using binary packages, simply run: `pkg install iocage`
+iocage is in the FreeBSD ports tree as sysutils/py-iocage.
+To install using binary packages, simply run: `sysutils/py-iocage`
```

Comment:
@jungle-boogie If you look at PKGNAME you will see it's py27-iocage.

@bartekrutkowski We still need a conflict on these, I don't intend for them to coexist.